hack/local-up-cluster.sh

```diff
@@ -115,16 +110,6 @@ if [ "${CLOUD_PROVIDER}" == "openstack" ]; then
     fi
 fi
 
-#set feature gates if using ipvs mode
```

Comment:
Why are there so many changes to this file? It seems unrelated?

Reply:
Must be a rebase error. I'll finalize this PR.